third_party/islet/setup.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -e
+
+ROOT=$(git rev-parse --show-toplevel)
+HERE=$ROOT/third_party/islet
+
+ISLET=$HERE/remote
+ISLET_SDK=$ISLET/sdk
+ISLET_INC=$HERE/include
+ISLET_LIB=$HERE/lib
+
+TARGET_HDR=$ISLET_SDK/include/islet.h
+TARGET_LIB=$ISLET/out/x86_64-unknown-linux-gnu/debug/libislet_sdk.so
+
+# Sync islet
+cd $HERE
+wget https://github.com/Samsung/islet/archive/refs/tags/certifier-v1.0-beta.tar.gz
+tar xf certifier-v1.0-beta.tar.gz
+mv islet-certifier-v1.0-beta $ISLET
+
+# Install rust
+$ISLET/scripts/deps/rust.sh
+
+# Build islet sdk (simulated version for x86_64)
+cd $ISLET_SDK && cargo build
+
+mkdir -p $ISLET_INC $ISLET_LIB
+cp $TARGET_HDR $ISLET_INC
+cp $TARGET_LIB $ISLET_LIB
```

Comment:
Please try to always use `cp -p` so that the file's timestamp is preserved. (Helps script debugging.)

Reply:
I attached `-p` option.